README.md

```diff
@@ -1 +1,69 @@
-# presto-checks-master
\ No newline at end of file
+# Presto Checks
+
+This project defines build automation for Presto that is meant to be run periodically / for a selection of commits in selected branches.
+
+## Requirements
+- bash
+- [aws cli](https://github.com/aws/aws-cli#installation)
+- access to https://api.travis-ci.org/ from the machine you're executing `trigger_checks_for_latest_build.sh`
+- `$REPO` - the slug of the repo where you're reading this, e.g. `ArturGajowy/presto-checks-master`
+- `$TOKEN` - Travis token obtained by executing `travis --token` in this repo's dir
```

Comment:
will do that after we agree on the final name of the repo. My proposal is `presto-checks`. @ilfrin, @losipiuk WDYT?